go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -288,7 +300,7 @@ func (bsnc *BaseSecondaryNetworkController) addLogicalPortToNetworkForNAD(pod *k
 		return err
 	}
 
-	if bsnc.doesNetworkRequireIPAM() && util.IsMultiNetworkPoliciesSupportEnabled() {
+	if util.IsMultiNetworkPoliciesSupportEnabled() {
 		// Ensure the namespace/nsInfo exists
 		addOps, err := bsnc.addPodToNamespaceForSecondaryNetwork(pod.Namespace, podAnnotation.IPs)
```

Comment:
Can you remind me why we need this to be called here on IPAM-less networks? I don't see the reason now.

Reply:
Right, this isn't needed. Good catch.

Done.